video-object-detection/.gitignore

```diff
@@ -0,0 +1,34 @@
+args.cfg
+
+/data/mnist/images/
```

Comment:
Absolute path names in a `.gitignore` seems strange to me.

Reply:
Yeah. That's a copy of the `.gitignore` in the deep learning server example